delta/migration/scripts/config/migration.conf

```diff
@@ -1,19 +1,33 @@
 migration {
+  # Configuration of the batches of messages to be inserted in postgres
```

Comment:
Should it be `PostgreSQL` instead of `postgres` to match the documentation?

Reply:
I think it should be consistent, yes, thanks 👍